### PR TITLE
feat: [CSRS-227] refactoring questionnaire

### PR DESCRIPTION
**Pull request by kkadaru**
# Description

This PR includes the following proposed change(s):

- CSRS-227 refactoring questionnaire

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by skiselev1 — COMMENTED** [review]
Thank you for cleaning some issues with scss! Very clean source code. I did not find questionnaire.html. Did you refactor it as well?

**Review by skiselev1 — APPROVED** [review]
Good job! I don't have any comments.

File: src/frontend/csrs-portal/src/app/components/questionnaire/questionnaire.component.ts [hunk 1]
```diff
@@ -3,132 +3,548 @@ import { AfterViewInit, Component } from '@angular/core';
...
-
-  // Question 1
-  q1Yes()
-  {
-    this.a1Yes = true; this.a1No = !this.a1Yes;
-  }
-  q1No()
-  {
-    this.a1No = true; this.a1Yes = !this.a1No;
+    this.bceIdLink = 'https://www.bceid.ca/';
```
Comment (NovaVic) [inline src/frontend/csrs-portal/src/app/components/questionnaire/questionnaire.component.ts:517]: FYI:
We may have to pass/load environment specific values here.
The link is not same for dev, test and prod.

**Review by NovaVic — CHANGES_REQUESTED** [review]
One minor modification request on dynamic/env specific BCeID link please. Thanks!

**Comment by kkadaru** [comment]
>

**Pull request merged**